huxley/api/tests/school/test_school_delegates.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+# Use of this source code is governed by a BSD License (see LICENSE).
+
+from huxley.api.tests import ListAPITestCase, UpdateAPITestCase
+from huxley.core.models import Assignment, Delegate, School
+from huxley.utils.test import (TestCommittees, TestCountries, TestSchools,
+                               TestUsers, TestAssignments, TestDelegates)
+
+
+class SchoolDelegateGetTestCase(ListAPITestCase):
+    url_name = 'api:school_delegates'
+    is_resource = True
+
+    def setUp(self):
```

Comment:
It looks like we're not modifying these objects in between test cases. In that case, we can use the class method, which is run only once (instead of before every test). This should make the test run a bit faster: 

https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.TestCase.setUpTestData

Furthermore, we don't have to make all of these objects class/instance members. For example, we only use `committee1` in this method, so it can just be a local variable instead of a `self.committee1` property.